pennylane/gradients/pulse_gradient.py

```diff
@@ -188,9 +224,14 @@ def _stoch_pulse_grad(tape, argnum=None, num_split_times=1, sampler_seed=None, s
             rule underlying the differentiation; also see details
         sample_seed (int): randomness seed to be used for the time samples in the stochastic
             parameter-shift rule
-        shots (None, int, list[int]): The device shots that will be used to execute the tapes outputted by this
-            transform. Note that this argument doesn't influence the shots used for tape execution, but provides
-            information about the shots.
+        shots (None, int, list[int]): The shots of the device used to execute the tapes which are
+            returned by this transform. Note that this argument does not *influence* the shots
+            used for execution, but informs the transform about the shots to ensure a compatible
+            return value structure.
+        use_broadcasting (bool): Whether to use broadcasting across the different sampled
+            splitting times. If ``False`` (the default), one set of modified tapes per
+            splitting time is created, if ``True`` only a single set of broadcasted, modified
+            tapes is created, increasing performance. This is only available on simulators.
```

Comment:
```suggestion
            tapes is created, increasing performance. This is only available on simulators and can speed up performance.
```

Reply:
This duplicates the statement before, no? :thinking:

Reply:
Oh right. Maybe just saying "..increasing performance on simulators"?